### DISCUSSION TITLE
Usage with Zapier

### DISCUSSION THREAD

**Discussion by xxczaki** [planetscale/discussion#179] [Q&A] [answered]
I would like to use PlanetscaleDB with Zapier, using their [Premium MySQL action](https://zapier.com/help/doc/how-get-started-mysql):

![CleanShot 2022-03-28 at 19 00 59](https://user-images.githubusercontent.com/22645979/160449980-ba64275d-d96f-409d-a570-8319e5856aa5.png)

Unfortunately, when I input everything expect for the SSL-related stuff, I get an error that no unsecured connections are allowed. I read https://docs.planetscale.com/concepts/secure-connections, but I don't really know what I should put in the SSL-related fields. Or maybe it's not possible to use PSDB with Zapier at all?

**Comment by jgreet** [comment, marked as answer]
Sorry, I don't see a way in Zapier's MySQL setup to enable SSL without providing the client certificate or a way to tell the client to use the system CA root store for verification. 

I don't think you'll be able to connect Zapier to PlanetScale as is. It may be worth asking them to add more SSL options.

**Comment by JordanSucher** [comment]
I had the same problem and figured out how to connect to Planetscale w Zapier.

Seems you can manually copy and paste from the SSL certificate that exists on your computer. Unsure of the specifics but see this line from documentation:

> PlanetScale uses certificates from a [Certificate Authority (CA)](https://en.wikipedia.org/wiki/Certificate_authority) that is part of the system root available on almost all platforms. We do this so that we can provide the easiest possible way for you to connect securely to PlanetScale (https://planetscale.com/docs/concepts/secure-connections)

What I did:
- Get my local ssl cert (on a mac, located at /etc/ssl/cert.pem)
- Open w text editor, copy into the "SSL CA Bundle" field, currently the 3rd and last SSL field
